src/java/org/apache/cassandra/index/sai/memory/TrieMemtableIndex.java

```diff
@@ -487,6 +477,13 @@ private ByteComparable encode(ByteBuffer input)
         return Version.current().onDiskFormat().encodeForTrie(input, indexContext.getValidator());
     }
 
+    private int getEndShardForBounds(AbstractBounds<PartitionPosition> bounds)
+    {
+        var position = bounds.right;
```

Comment:
Can you use the explicit type instead of `var`? AFS prohibits using `var` unfortunately.